R/drive_cp.R

```diff
@@ -11,6 +11,7 @@
 #' @template name
 #' @templateVar name file
 #' @templateVar default Defaults to "Copy of `FILE-NAME`".
+#' @param ... Parameters to pass along to the API query.
```

Comment:
Should say a bit more about what this might be? I.e. whatever you might accomplish re: metadata with `...`.